### ISSUE TITLE
Deleted columns still being shown in PostgreSQL metadata. 

### ISSUE THREAD

**Issue by synox** [metabase/metabase#6108]
We renamed a column in a simple table without foreign keys. Now the question shows the error: 

    ERROR: column accounts.nach_pb_bezeichnung does not exist Hint: 
    Perhaps you meant to reference the column "accounts.nach_bp_bezeichnung". Position: 2374

 - Metabase version v0.26.1, Built on 2017-09-26
- Postgresql: version 9.5
- Metabase hosting environment: Dockerfile on rhel 8 with Openshift 3.5
- Metabase internal database: Postgres

- *Repeatable steps to reproduce the issue*
Rename Column and create new question. 

Might be related to: https://github.com/metabase/metabase/issues/5814 

```
Oct 23 16:08:19 WARN metabase.query-processor :: Query failure: ERROR: column accounts.nach_pb_bezeichnung does not exist
Hint: Perhaps you meant to reference the column "accounts.nach_bp_bezeichnung".
Position: 2438
["query_processor$assert_query_status_successful.invokeStatic(query_processor.clj:203)"
"query_processor$assert_query_status_successful.invoke(query_processor.clj:196)"
"query_processor$run_and_save_query_BANG_.invokeStatic(query_processor.clj:235)"
"query_processor$run_and_save_query_BANG_.invoke(query_processor.clj:229)"
"query_processor$fn__28243$process_query_and_save_execution_BANG___28248$fn__28249.invoke(query_processor.clj:272)"
"query_processor$fn__28243$process_query_and_save_execution_BANG___28248.invoke(query_processor.clj:258)"
"api.dataset$fn__30133$fn__30136.invoke(dataset.clj:63)"
"api.common.internal$do_with_caught_api_exceptions.invokeStatic(internal.clj:248)"
"api.common.internal$do_with_caught_api_exceptions.invoke(internal.clj:243)"
"api.dataset$fn__30133.invokeStatic(dataset.clj:54)"
"api.dataset$fn__30133.invoke(dataset.clj:54)"
"middleware$enforce_authentication$fn__29522.invoke(middleware.clj:122)"
"api.routes$fn__43823.invokeStatic(routes.clj:61)"
"api.routes$fn__43823.invoke(routes.clj:61)"
"routes$fn__44402$fn__44403.doInvoke(routes.clj:75)"
"routes$fn__44402.invokeStatic(routes.clj:71)"
"routes$fn__44402.invoke(routes.clj:71)"
"middleware$log_api_call$fn__29621$fn__29623.invoke(middleware.clj:330)"
"middleware$log_api_call$fn__29621.invoke(middleware.clj:329)"
"middleware$add_security_headers$fn__29571.invoke(middleware.clj:245)"
"middleware$bind_current_user$fn__29526.invoke(middleware.clj:142)"
"middleware$maybe_set_site_url$fn__29575.invoke(middleware.clj:268)"]
```

**Comment by chharland** [comment]
I have the same issue, but on v0.26.2 and Posgres 9.6. Additionally, if I drop the offending table and force a sync of the db the entire schema associated with the table is removed. Then I need to sync again to get all the other tables back.

**Comment by LienM** [comment]
Same issue. v0.26.2, Postgres 9.6. I've "overridden" it by manually excluding the column from the data model.

**Comment by senior** [comment]
This issue was fixed by https://github.com/metabase/metabase/pull/6601 and will be included in the next release (probably 0.28.0 RC1).
